components/p3a/nitro_utils/cose.cc

```diff
@@ -36,13 +36,13 @@ bool ConvertCoseSignatureToDER(const std::vector<uint8_t>& input,
   }
 
   BIGNUM* r_comp = BN_bin2bn(input.data(), kSignatureComponentSize, nullptr);
-  DCHECK(r_comp);
+  CHECK(r_comp);
   BIGNUM* s_comp = BN_bin2bn(input.data() + kSignatureComponentSize,
                              kSignatureComponentSize, nullptr);
-  DCHECK(s_comp);
+  CHECK(s_comp);
 
   ECDSA_SIG* ecdsa_sig = ECDSA_SIG_new();
```

Comment:
looking at how these functions are used in the browser, i think it's better to stick to `if`s instead of `CHECK`s. Otherwise, we are essentially `CHECK`ing the fact of successful memory allocation, which we shouldn't do